### ISSUE TITLE
Add UART I/O

### ISSUE THREAD

**Issue by larsbrinkhoff** [larsbrinkhoff/xForth#52]
Add library for using an UART on targets that have it.

**Comment by TG9541** [comment]
An abstract `TX!` and `?RX` should do the job. These can even be used for buffered I/O if needed.

**Comment by larsbrinkhoff** [comment]
Right, something like that.  I'm thinking I might as well adopt the classical words `EMIT`, `KEY`, and `KEY?`.

An optional extension on top of this would be to have a target resident interpreter.

**Comment by larsbrinkhoff** [comment]
If multitasking (#48) is loaded, `KEY` should `PAUSE`.
